src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -63,6 +64,11 @@ public IWorkspaceModel CurrentWorkspaceModel
             }
         }
 
+        public IEnumerable<NotificationMessage> Notifications
```

Comment:
seems weird that this is sent to all extensions...

Reply:
can we talk about alternative solutions?

Reply:
I'm all ears. This is one way I could think of since the Notification extension needs to be linked to the DynamoLogger some how.